application/dashboards/ha-proxy-dashboard-example.json

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
@MichaelLi202112 can we have some simple example this seems way too big?

Reply:
followed in sessions below